ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/defendant/CCDRespondent.java

```diff
@@ -112,4 +116,8 @@ public boolean hasStatementOfTruth() {
         return responseDefendantSOTSignerName != null
             || responseDefendantSOTSignerRole != null;
     }
+
+    public void setResponseDeadline(LocalDate responseDeadline) {
```

Comment:
Do we really need this ? We can use ` @Builder(toBuilder = true)` and then builder object where necessary.

Reply:
hey, yes thank you for your feedback, will change this.